bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java

```diff
@@ -480,10 +485,17 @@ protected String executeSingle(ResourceBundle language, String[] labelFragments,
                     if (oldState.equals(newState)) {
                         String template = language.getString("state_already_singular");
                         String cmdName = "state_" + command.toString().toLowerCase();
-                        String stateText = language.getString(cmdName);
+                        String stateText = null;
+                        try {
+                            stateText = language.getString(cmdName);
+                        } catch (Exception e) {
+                            stateText = language.getString("state_current");
```

Comment:
There are (already were) quite some other strings for which there is no constant defined in here - I did it similar to the state_already_* ones. But yes, all of these strings could be defined as constants throughout this class.